lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -467,12 +454,11 @@ export class UserAgentApplication {
                 // Push popup window handle onto stack for tracking
                 WindowUtils.trackPopup(popUpWindow);
             } catch (e) {
-                this.loginInProgress = false;
-                this.acquireTokenInProgress = false;
+                this.cacheStorage.setItem(TemporaryCacheKeys.INTERACTION_STATUS, RequestStatus.CANCELLED);
```

Comment:
Does setting to cancelled here do anything? Doesn't this get wiped out anything during removeAcquireTokenEntries?

Reply:
Resetting interaction status since we bail out from here with a reject. removeAcquireTokenEntries() checks for this status to remove the entires for a request. So setting this to 'Cancelled' or 'Completed' is necessary.